bundles/org.openhab.binding.unifi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -41,7 +41,7 @@
 
 		<label>UniFi Network</label>
 		<description>A UniFi Network</description>
-
+		<semantic-equipment-tag>NetworkSwitch</semantic-equipment-tag>
```

Comment:
This is a logical network within the UniFi configuration, not a network switch.

Reply:
Yes. Do you have another suggestion?

Reply:
No.

Reply:
Ok. Removing all unclear tags from this binding..